packages/common/src/groups/HubGroups.ts

```diff
@@ -84,6 +85,10 @@ export async function enrichGroupSearchResult(
   result.links.thumbnail = getGroupThumbnailUrl(requestOptions.portal, group);
   result.links.self = getGroupHomeUrl(result.id, requestOptions);
   result.links.siteRelative = `/teams/${result.id}`;
+  result.links.workspaceRelative = getRelativeWorkspaceUrl(
+    result.type,
```

Comment:
In this case you should just hardcode `"group"` since this function is only used for group search results. I'm not sure what `group.type` will give you, but I suspect it won't work. The reason typescript doesn't give you an error is that the first argument is labeled as a `string` instead of an `HubEntityType` (something another PR should definitely consider changingg

Reply:
Updated it to `"Group"` as it would expect.

Reply:
Good catch!